src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -69,13 +72,11 @@ export const ConversationRoute: React.FC<ConversationRouteProps> = props => {
               selectedConversationID={me.conversation.internalID}
             />
           </Media>
-          {/* @ts-ignore */}
           <Conversation conversation={me.conversation} />
           <Details
-            // @ts-ignore
-            conversation={me.conversation as any /** FIXME: Correct type */}
-            display={["none", null, null, null, "flex"]}
-            width={["100%", "376px"]}
+            conversation={me.conversation}
+            display="flex"
```

Comment:
Do we need to pass `flex` down to the `Details` component? It'd be nice if that could handle whatever display it needs on its own.